src/modules/peek/Peek.UI/MainWindow.xaml.cs

```diff
@@ -33,6 +33,7 @@ public MainWindow()
         {
             InitializeComponent();
             this.Activated += PeekWindow_Activated;
+            Title = "Peek";
```

Comment:
Probably you can use `ResourceLoader.GetForViewIndependentUse().GetString("AppTitle")` instead of the hardcoded one. There is a AppTitle string in resources, but it's not used anywhere.

Reply:
True. When I tried it, it didn't work. I could try again..